pkgs/applications/backup/cdpfgl/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, pkgconfig, intltool, glib, sqlite, jansson, 
+  libmicrohttpd, curl}:
+
+stdenv.mkDerivation rec {
+  name = "cdpfgl-${version}";
+  version = "0.0.11"; 
+
+  buildInputs = [ pkgconfig intltool glib sqlite jansson libmicrohttpd curl ]; 
+  outputs = [ "out" "dev" "man" ];
+
+  src = fetchurl {
+    url = "http://cdpfgl.delhomme.org/download/releases/${name}.tar.xz";
+    sha256 = "16ablsb2hnni0d28y73w1gk8j8gsmyd5rcjxx4n7zdv5ph8fz5bd";
+  };
+  
+  meta = with stdenv.lib; {
+    homepage = https://github.com/dupgit/sauvegarde;
+    description = "Continuous data protection for GNU/Linux (cdpfgl).";
+    maintainers = "Olivier Delhomme";
+    license = licenses.gpl3;
+    platforms = platforms.linux;
```

Comment:
Do you want to maintain this package?

Reply:
Hi, 

I'm not sure of what it means but as I'm already the core developer of the project I would say why not. Do I have to fill something somewhere ?

Thanks.

Reply:
Just add yourself as `dupbit` to `lib/maintainers.nix` and add yourself to this package:

```
  meta = {
    maintainers = [ dupgit ];
  };
```

As a consequence you will receive emails from hydra, when something regarding the build went wrong.  We may consult you, when people are making changes to this package like applying updates. I have also summarized all information here: https://nixos.wiki/wiki/Get_Involved#Becoming_a_Nixpkgs_maintainer

Reply:
Ok. Thanks.